cdap-distributions/src/javadoc/stylesheet.css

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2014 Cask Data, Inc.
+ * Copyright © 2015 Cask Data, Inc.
```

Comment:
Shouldn't this be [...] © 2014-15 Cask Data [...]?
I thought the rule was single year if it is a new file or document.